oktapam/resource_project.go

```diff
@@ -147,6 +152,7 @@ func resourceProjectCreate(ctx context.Context, d *schema.ResourceData, m interf
 		SSHSessionRecording:    getBoolPtr(attributes.SSHSessionRecording, d, false),
 		GatewaySelector:        getStringPtr(attributes.GatewaySelector, d, false),
 		SSHCertificateType:     getStringPtr(attributes.SSHCertificateType, d, false),
+		UserOnDemandPeriod:     getIntPtr(attributes.UserOnDemandPeriod, d, false),
```

Comment:
Is UserOnDemandPeriod only need to be passed if create_server_users flag is set to true - [?](https://okta.slack.com/archives/C03S22Y5AH1/p1663083292332599). I am not sure on this.

Reply:
I believe they're independent. Looking throughout the code UserOnDemandPeriod and CreateServerUser are applied independently.